### PR TITLE
Homework 1 checkin

### PR DESCRIPTION
**Pull request by jasong-peakbagger**
I tried so hard to get rid of these last few errors. Simply ran out of time. 

Not sure what to do with module.exports - how is it undefined?
Not sure what to do around line 104. I have no idea what it's doing. :(
-JG

### CODE REVIEW

**Comment by jamdon** [comment]
Hi Jason. For variables that haven't been defined but aren't simple var declarations, you can use a "globals" block in your configuration file (see the eslint configuration documentation to see what that looks like.) As for the semi-colons, inspect that code carefully to see what it's doing, as there are, simply, some semi-colons missing. Alternatively you can watch the homework review recording I've posted in the recordings section of the course github wiki. Let me know if you want this graded as-is or you can push an update to your forked repo and let me know if you want it re-reviewed.

**Comment by jasong-peakbagger** [comment]
Got it! Please re-review this for the correction. Thanks for the tip with use of Globals in the lint file and the semicolons needed.

File: map-utils.js [hunk 1]
```diff
@@ -1,126 +1,157 @@
...
+  if (!placeObj.formatted_address) {
+    return null;
+  };
+
+  var htmlAddress = placeObj.adr_address;
+  var firstPart = htmlAddress.substr(0, htmlAddress.indexOf('<'));
+  var addressContainer = window.document.createElement('span');
+  addressContainer.innerHTML = htmlAddress;
+  var childNodes = Array.prototype.slice.call(addressContainer.children, 0);
+  if (childNodes.length == 0) {
```
Comment (jamdon) [inline map-utils.js]: Upon closer inspection, it looks like you're missing a rule required in the assignment:

_*\* always use strict equality/inequality**_

**Comment by jamdon** [comment]
The corrections look good, but I found one more issue of a missing linting rule. See comment inline for details.

**Comment by jasong-peakbagger** [comment]
It's in there @ line 31 of the lint file.

Unless, of course, I dont have it in correctly. I'll double check.  I did
notice that with the way I have the rule in place it forced me to have a
"use strict" line at almost each function.

-JG

**Comment by jamdon** [comment]
Did you update your github repo? I did a git pull on your repo, but line 31
of that lint file is the strict rule.

thanks,

James

File: map-utils.js (hunk 1 above)
Comment (jasong-peakbagger) [inline map-utils.js]: This is corrected now. I wasn't catching on that I needed the option "global" for the strict rule. Both files were update and use-strict is now global. All errors were cleared.

**Comment by jamdon** [comment]
I still don't see the "eqeqeq" linting rule in your configuration file. Once you add it you'll find some additional linting errors.

**Comment by jasong-peakbagger** [comment]
Geez. I now get it. When I seen "always use strict equality/inequality". I did a search on "use strict" and didn't pick up on the "equality/inequality". Hopefully this does it.

**Comment by jamdon** [comment]
Make sure you don't close the p.r. yourself or I may not see that you've updated it.

**Comment by jamdon** [comment]
Looks good. 👍

**Pull request closed without merging**